README.md

```diff
@@ -40,6 +32,47 @@ mvn clean install
 
 This will build the artifacts after validating all checks including unit tests and place it in the deployer/target directory.
 
+
+## Installation
+
+* Build the project.
+* Login cf and select the space that Ocopea will deploy to.
```

Comment:
I would say: "Issue a "cf login" and select the space that Ocopea will deploy to.

Reply:
Fixed